third_party/docfx-doclet-143274/src/main/java/com/microsoft/build/YmlFilesBuilder.java

```diff
@@ -66,19 +83,32 @@ public boolean build() {
     processor.process();
 
     //  write to yaml files
-    FileUtil.dumpToFile(processor.projectMetadataFile);
+    if (disableLibraryOverview) {
+      FileUtil.dumpToFile(processor.projectMetadataFile);
+    }

```

Comment:
Why is this behavior dependent on the disableLibraryOverview setting?

Reply:
This was the logic to overwrite the current existing overview with the new overview file, assuming the `disable-libraryOverview` flag is false). Let me know though if you have a different approach in mind for doing so.